Cart Flow/Custom Views/Scenes/ListSelectVC.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  ViewController.swift
+//  Cart Flow
+//
+//  Created by Scott Quintana on 8/3/20.
+//  Copyright © 2020 Scott Quintana. All rights reserved.
+//
+
+import UIKit
+import CoreData
+
+class ListSelectVC: UIViewController {
+
+    let context = (UIApplication.shared.delegate as! AppDelegate).persistentContainer.viewContext
+    let tableView = UITableView()
+    var shoppingLists: [ShoppingList] = [].sorted {
+        $0.lastUpdate?.compare($1.lastUpdate!) == .orderedDescending
+    }
```

Comment:
I believe this'll just sort the initial `shoppingLists`, which is an empty array. You could remove the `.sorted` call here.